public/app/features/explore/state/actionTypes.ts

```diff
@@ -43,6 +43,7 @@ export enum ActionTypes {
   ToggleTable = 'explore/TOGGLE_TABLE',
   UpdateDatasourceInstance = 'explore/UPDATE_DATASOURCE_INSTANCE',
   ResetExplore = 'explore/RESET_EXPLORE',
+  SetInitialQueries = 'explore/SET_INITIAL_QUERIES',
```

Comment:
Any chance to avoid setter actions?

Reply:
I'm assuming you are referring to naming convention here? I'll update the name to make it not bound to performed state change, but queries import is a de facto setter.